pages/osx/hdiutil.md

```diff
@@ -0,0 +1,19 @@
+# hdiutil
+
+> Utility to create and manage disk images.
+
+- Mount an image:
+
+`hdiutil attach {{filename}}`
```

Comment:
All of the tokens should be snake_case, like file_name, volume_name.

Reply:
I think `filename` is ok as-is - I believe we've done `path/to/filename` before though.

Reply:
Yeah ok, I see a lot of `filename`s. But we should change dirname to path/to/directory though. There are no other `dirname`s